.github/ISSUE_TEMPLATE/demand.md

```diff
@@ -13,9 +13,9 @@ Field | Value
 --- | ---
 Previous Stage | ...
 Figma Link | [figma.com/...](___)
```

Comment:
In this line, use `/** figma.com/... */ Link here` might be better.

Reply:
Good point. I will change it to a simple `__Figma_Link_Here__`.